File: Src/Visual Studio/SDK/C#/Eruru.ChatRobotRPC For Shared Project/Enums/ChatRobotMessageType.cs
namespace Eruru.ChatRobotRPC {

	/// <summary>
	/// 消息类型
	/// </summary>
	public enum ChatRobotMessageType {

		/// <summary>
		/// 好友
		/// </summary>
		Friend = 1,
		/// <summary>
		/// 群临时
		/// </summary>
		GroupTemp = 2,
		/// <summary>
		/// 讨论组临时
		/// </summary>
		DiscussTemp = 3,
		/// <summary>
		/// 网页临时
		/// </summary>
		WebpageTemp = 4,
		/// <summary>
		/// 好友验证回复
		/// </summary>
		FriendVerificationReply = 5,
		/// <summary>
		/// 群
		/// </summary>
		Group = 6,
		/// <summary>
		/// 讨论组
		/// </summary>
		Discuss = 7,

	}

}

File: Src/Intelij IDEA/SDK/Java/Eruru.ChatRobotRPC/src/main/java/org/eruru/chatrobotrpc/enums/ChatRobotMessageType.java
package org.eruru.chatrobotrpc.enums;

/**
 * 消息类型
 */
public enum ChatRobotMessageType {

	/**
	 * 好友
	 */
	friend (1),
	/**
	 * 群临时
	 */
	groupTemp (2),
	/**
	 * 讨论组临时
	 */
	discussTemp (3),
	/**
	 * 网页临时
	 */
	webpageTemp (4),
	/**
	 * 好友验证回复
	 */
	friendVerificationReply (5),
	/**
	 * 群
	 */
	group (6),
	/**
	 * 讨论组
	 */
	discuss (7);

	private final int value;

	ChatRobotMessageType (int value) {
		this.value = value;
	}

	/**
	 * 获取枚举对应的整形
	 */
	public int getValue () {
		return value;
	}

	/**
	 * 通过整形获取对应的枚举
	 */
	public static ChatRobotMessageType get (int value) {
		for (ChatRobotMessageType item : ChatRobotMessageType.values ()) {
			if (item.value == value) {
				return item;
			}
		}
		return null;
	}

}